raiden/network/transport/matrix/layer.py

```diff
@@ -85,27 +85,27 @@ def __init__(self, config: Dict[str, Any]):
 
                 light_client_transports.append(light_client_transport)
```

Comment:
Can we just use self._light_clients.append() insteadof create a new list and assign it to the member of the class?

Reply:
done